src/Writers/LaTeXWriter.jl

```diff
@@ -533,7 +533,7 @@ function latexinline(io::IO, md::Markdown.Image)
             # URLs starting with a / are assumed to be relative to the document's root
             normpath(lstrip(md.url, '/'))
         else
-            normpath(joinpath(dirname(io.filename), md.url))
+            normpath(dirname(io.filename) * "/" * md.url)
```

Comment:
I guess there could also be `\\` in `dirname(io.filename)` and `md.url`? Maybe safest to do this:
```suggestion
            normpath(replace(dirname(io.filename) * "/" * md.url, "\\" => "/"))
```